### DISCUSSION TITLE
Is it possible to parameterize a resource with a minimum usage time in Frepple?

### DISCUSSION THREAD

**Discussion by salvastp** [frePPLe/frepple#671] [Q&A]
Hi everyone,

I’m running some tests with Frepple and working on a resource that has a particular constraint: to be cost-effective, it must be used for a minimum number of continuous hours due to the high costs associated with starting it up (1 hour of heating) and cleaning it. Therefore, we need to accumulate manufacturing orders before activating it, as it cannot be used indiscriminately. How should I parameterize this type of resource in the system to ensure that it is scheduled only when the required production minimums are met, optimizing its profitability?

Thanks in advance for your help!

**Comment by jdetaeye** [comment]
There is no standard functionality to achieve this.

Your options are:
- Do nothing, and let the algorithm generate a schedules without aggregation and that shows "holes" idle periods in the plan. The planner can then judge manually every day whether there are sufficient manufacturing orders accumulated to run a batch that day (which can also depend on operator availability, urgency of the manufacturing orders, loading on other resources, etc...).   
- In some cases, it can appropriate to enforce the predefined accumulation by using a calendar: eg "resource X only works on mondays and thursday".
- Use some postprocessing logic to do the aggregation.

**Reply by salvastp** [reply]
Thanks for the response.

My approach to the problem has been to create calendars where the resource can only be used one day a week, hoping that the system will try to accumulate as many manufacturing orders as possible for that day. I know it’s not the most optimal solution, but I think it might work in my case.

Thanks again for the guidance!
